src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -365,8 +369,23 @@ private void OpenIndexes()
 
                     exceptionAggregator.Execute(() =>
                     {
-                        var index = Index.Open(indexId, _documentDatabase);
-                        _indexes.Add(index);
+                        try
+                        {
+                            var index = Index.Open(indexId, _documentDatabase);
+                            _indexes.Add(index);
+                        }
+                        catch (Exception e)
+                        {
+                            // TODO arek: I think we can ignore auto indexes here, however for static ones try to retrieve names
+                            var fakeIndex = new FaultyInMemoryIndex(indexId);
```

Comment:
Why are we passing just the id? Why not the name as well?
Don't we have a name here?